docs/source/blocks/blocks-core/teaser.md

```diff
@@ -0,0 +1,152 @@
+    "keywords": "Volto, React, blocks, teaser, Plone"
+---
+
+(teaser-block)=
+
+# Teaser Block
+
+Volto comes with a Teaser block that given a content object as target is able to pull information from it and show it in the block.
+By default, this information is the `title`, `description`, `preview_image` and shows them in a summary (Image on left, title, description) way.
+
+## Blocks settings
+
+The Teaser has the following settings:
+
+Target
+: The target is either an existing content item in your Plone site that can be selected by clicking the list icon, or an external URL that can be typed into the text field and saved by clicking the right arrow when it appears after you start typing.
+
+Title
+: The title is the title of the content item.
+
+Head title
+: The head title is a heading that appears above the title.
+
+Description
+: The description is plain text that summarizes or describes the content item.
+
+Image override
+: The image override is either an existing image in your Plone site that can be selected by clicking the list icon, or an external URL of an image that can be typed into the text field and saved by clicking the right arrow when it appears after you start typing.
+
+## Style Wrapper Properties
+
+The Teaser has a styling property to align the image. The image can be aligned left (default), right and top.
+
+The styling wrapper properties can be extended using: {ref}`blocks-style-wrapper-label`
+
+## Variations
+
+By default, the Teaser have a default variation, but this can be extended using block variations
+
+The variations can be extended using: {ref}`extensions-block-variations`
+
+## Data Adapter
+
+The Teaser has a data adapter function that allows you to tap into the changes in the settings, and being able to issue changes in other settings fields.
+This is valuable in the Teaser block because it saves an internal cache of the target element.
+If you select the target, these values are updated. Also, when you update the target, these values remain (default behavior), but you can issue another behavior.
+
+The data adapter function is defined in the block's settings `dataAdapter`, so you can override it and add your own, if required.
+This is the default adapter (you should stick to his signature):
+
+```js
+import { isEmpty } from 'lodash';
+
+export const TeaserBlockDataAdapter = ({
+  block,
+  data,
+  id,
+  onChangeBlock,
+  value,
+}) => {
+  let dataSaved = {
+    ...data,
+    [id]: value,
+  };
+  if (id === 'href' && !isEmpty(value) && !data.title && !data.description) {
+    dataSaved = {
+      ...dataSaved,
+      title: value[0].Title,
+      description: value[0].Description,
+      head_title: value[0].head_title,
+    };
+  }
+  onChangeBlock(block, dataSaved);
+};
+
+```
+
+## Custom components in registry
+
+The Teaser block looks up for a couple of components in the registry, that you can provide in case that you want to modify the default behavior.
```

Comment:
```suggestion
The Teaser block looks up a couple of components in the registry that you can provide in case you want to modify the default behavior.
```